yt_dlp/extractor/googledrive.py

```diff
@@ -56,24 +57,17 @@ class GoogleDriveIE(InfoExtractor):
     }, {
         'url': 'https://drive.usercontent.google.com/download?id=0ByeS4oOUV-49Zzh4R1J6R09zazQ',
         'only_matching': True,
+    }, {
+        # mp3 (format 50) tests
+        # Royalty Free music from 1922
+        # My Buddy by Henry Burr; Gus Kahn; Walter Donaldson
+        # https://archive.org/details/78_my-buddy_henry-burr-gus-kahn-walter-donaldson_gbia0015627a/My+Buddy+-+Henry+Burr+-+Gus+Kahn+-+Walter+Donaldson.flac
+        'url': 'https://drive.usercontent.google.com/download?id=1IP0o8dHcQrIHGgVyp0Ofvx2cGfLzyO1x',
+        'only_matching': False,
```

Comment:
please actually run the test and add in the required fields

Reply:
```
$cpython3 -m pip install git+https://github.com/WyohKnott/yt-dlp@patch-1

$ python devscripts/run_tests.py GoogleDrive  
Running ['pytest', '-Werror', '--tb=short', 'test/test_download.py::TestDownload::test_GoogleDrive']
========================================================================= test session starts ==========================================================================
platform linux -- Python 3.12.3, pytest-7.4.4, pluggy-1.3.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /home/wyoh/yt-dlp
configfile: setup.cfg
plugins: anyio-4.3.0
collected 1 item                                                                                                                                                       

test/test_download.py::TestDownload::test_GoogleDrive PASSED                                                                                                     [100%]

========================================================================== 1 passed in 2.76s ===========================================================================
```

> add in the required fields

I've searched in the contributing guide, but I am not sure what required fields you are mentioning.

Reply:
It's only running the first test (i.e. not the one you added). Try `GoogleDrive_1`

Reply:
Ok we're good:

```
$ python3 -m pip install git+https://github.com/WyohKnott/yt-dlp@patch-1

Running ['pytest', '-Werror', '--tb=short', 'test/test_download.py::TestDownload::test_GoogleDrive_1']
========================================================================= test session starts ==========================================================================
platform linux -- Python 3.12.3, pytest-7.4.4, pluggy-1.3.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /home/wyoh/yt-dlp
configfile: setup.cfg
plugins: anyio-4.3.0
collected 1 item                                                                                                                                                       

test/test_download.py::TestDownload::test_GoogleDrive_1 PASSED                                                                                                   [100%]

========================================================================== 1 passed in 4.48s ===========================================================================
```